packages/tasit-sdk/package.json

```diff
@@ -25,7 +25,7 @@
     "test": "npm run setup:local && npm run mocha",
     "test:forked-testnet": "npm run setup:forked-ropsten && npm run mocha",
     "test:testnet": "npm run setup:ropsten && npm run mocha",
-    "mocha": "npx mocha src/*.test.js  --require @babel/register --file src/testHelpers/mochaSetup.js --no-timeouts",
+    "mocha": "npx mocha \"src/**/*.test.js\"  --require @babel/register --file src/testHelpers/mochaSetup.js --no-timeouts",
```

Comment:
Were we missing some tests or including extra files before?

Reply:
No, not really. Until now we had the `Decentraland.test.js` on the root dir.

Reply:
Ah yeah, that's what I figured once I saw that you added a new test file. Resolving this.